cli/main.c

```diff
@@ -275,20 +285,37 @@ static int bw(int argc, char **argv)
 		struct switchtec_dev *dev;
 		unsigned meas_time;
 		int verbose;
+		enum switchtec_bw_type bw_type;
 	} cfg = {
 		.meas_time = 5,
+		.bw_type = SWITCHTEC_BW_TYPE_RAW,
 	};
+
+	const struct argconfig_choice types[] = {
+		{"RAW", SWITCHTEC_BW_TYPE_RAW, "Get the raw bandwidth"},
+		{"PAYLOAD", SWITCHTEC_BW_TYPE_PAYLOAD, "Get the payload bandwidth"},
+		{}
+	};
+
```

Comment:
Don't repeat yourself. Make this 'static const', file scope and a more descriptive name.

Reply:
Thank you very much! I updated my codes.